src/components/PlioQuestion.vue

```diff
@@ -32,6 +32,7 @@
                       name="options"
                       v-model="selectedOption"
                       :value="option"
+                      :id="option"
```

Comment:
Added `id` just so I could reference it in the code and select that radio button. This is needed because the option that was selected by the user in an older session, needs to be shown when a question is revisited.

Reply:
But is it different from `value`?

Reply:
No it isn't. I thought it's easier to select an element using an `id` but then I was able to do the same thing with using it's `value` so I'll remove the `id` here.